templates/primary-secondary/test/primary-secondary.visual-diff.js

```diff
@@ -0,0 +1,33 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-template-primary-secondary', function() {
+	const visualDiff = new VisualDiff('primary-secondary', __dirname);
+	let browser;
+	let page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+	});
+
+	after(() => browser.close());
+
+	describe('desktop', function() {
+		before(async() => {
+			await page.setViewport({width: 930, height: 930, deviceScaleFactor: 2});
+		});
+
+		[
+			{ name: 'normal', selector: '#normal', fileName: 'primary-secondary-desktop.visual-diff.html' },
+			{ name: 'larger-than-viewport-height', selector: '#larger-than-viewport-height', fileName: 'primary-secondary-desktop-large.visual-diff.html' }
+		].forEach((entry) => {
+			it(entry.name, async function() {
+				await page.goto(`${visualDiff.getBaseUrl()}/templates/primary-secondary/test/${entry.fileName}`, {waitUntil: ['networkidle0', 'load']});
+				await page.bringToFront();
+				const rect = await visualDiff.getRect(page, entry.selector);

```

Comment:
Nit: since the pages only have the one element in them you could remove the `id` and simplify this to:
`visualDiff.getRect(page, 'd2l-template-primary-secondary')` (using an element selector).